packages/yoga/src/Datepicker/web/Datepicker.jsx

```diff
@@ -232,21 +232,21 @@ function Datepicker({
   };
 
   const renderInput = () => {
-    if (!startD && !endD) {
+    if (!startDate && !endDate) {
       return (
         <InputPlaceholder disabled={disabled}>
           {placeholder ?? `Select Date`}
         </InputPlaceholder>
       );
     }
 
-    const dateFormat = 'MMM D, YYYY';
+    const dateFormat = 'MMM d, yyyy';
```

Comment:
Are we considering handling localization in this component? I'm asking it because it can be inconsistent if on my whole page I'm formatting the dates by the user's country and this one is in a different format.

Reply:
As discussed with @tomaskeong, the team has agreed to follow the way it is, and we'll work to evolve this component in the future. Because of that, I'm opening an issue https://github.com/gympass/yoga/issues/550. So, feel free to comment and collaborate about that.